src/pages/changelog/2025-03-26.mdx

```diff
@@ -0,0 +1,11 @@
+---
+date: 2025-03-26
```

Comment:
What did you base the date on? Think we can just set it to "today"

Reply:
Ah ok, I based on [this commit](https://github.com/criipto/criipto-signatures-sdk/commit/93bfe3cc641611bf1c4e2f68da9a423cb552b8d0).

Reply:
Changed the date to today.